### ISSUE TITLE
Add qx create option to avoid all cursor control

### ISSUE THREAD

**Issue by derrell** [qooxdoo/qooxdoo-compiler#179]
At present, the wizard for `qx create` attempts to overwrite each question on the screen with the next question, i.e., there are a lot of cursor control commands being written to the screen. It would be nice to have an option that eliminated all of the cursor control output so that the wizard would work nicely on a dumb terminal (or in an emacs shell). With this option enabled, each question would appear on a line, the user would respond and press Enter, which would move the cursor to the next line where the subsequent question would be presented.

**Comment by cboulanger** [comment]
Looks like [inquirer](https://www.npmjs.com/package/inquirer), the npm library that we use for user interaction, doesn't provide any option for nicely degrading on dumb terminals. Need to do some research on alternatives that have better dumb terminal support, maybe one of the following?
- https://www.npmjs.com/package/promptly
- https://www.npmjs.com/package/prompts
- https://www.npmjs.com/package/readline-sync
- https://www.npmjs.com/package/prompt-sync
- https://www.npmjs.com/package/enquirer

**Comment by johnspackman** [comment]
Surely it would be easy to just write an `ask()` method that decides whether to use cursor based or readline?

**Comment by cboulanger** [comment]
Yes, but we have multiple-choice questions that would need to be manually translated into simple prompt, which is a chore I'd rather avoid :-)

**Comment by oetiker** [comment]
how about just picking the default if the thing is called without cursor control ... I guess the answers can be supplied on the commandline too

**Comment by cboulanger** [comment]
@oetiker Yes, you always have the option to supply all information via CLI parameters. But I think @derrell wants the interactivity without the cursor control :-)

**Comment by cboulanger** [comment]
@derrell Is this still a problem that seriously affects your workflow? Or is the CLI params option enough for your needs?

**Comment by derrell** [comment]
`qx create` is used infrequently enough that if it's a huge amount of work to implement this, it's reasonable for the user (me, in this case) to open up a separate window (outside of emacs) to run the command. OTOH, I don't know that I've seen many other apps that do cursor control that don't provide a means of disabling it. From my perspective, this certainly need not be a blocker to first release, but should be "fixed" eventually rather than being closed.
